docs/brief-install-vagrant-dev.md

```diff
@@ -105,3 +105,15 @@ HOST_PORT 9971
 
 (and halt and restart the Vagrant guest), then you would visit
 http://localhost:9971/ to connect to your development server.
+
+If you wish to set a different host IP address for Vagrant to use
```

Comment:
Maybe this should say "If you'd like to be able to connect to your development environment from other machines than the VM host, ..."